src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -47,7 +47,6 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="Microsoft.DiaSymReader.Native" />
     <NuGetPackageToIncludeInVsix Include="Microsoft.Win32.Primitives" />
-    <!-- Even though the VS process includes both of these, we need to also include them in the VSIX for the compiler processes -->
```

Comment:
I don't think so -- VS may or may not include a lot of things in this list, but we should always give our own copy, just in case it's a different version.